plugins/unmanaged_files/unmanaged_files_inspector.rb

```diff
@@ -90,7 +90,11 @@ def extract_unmanaged_files(files, trees, excluded, store_name)
     archive_path = File.join(store_path, "files.tgz")
     @system.create_archive(files.join("\0"), archive_path, excluded)
 
+    extracted_count = files.length
+
     trees.each do |tree|
+      phrase = Machinery::pluralize(extracted_count, "file or tree", "files and trees")
```

Comment:
So you'd prefer

```ruby
      Machinery::Ui.progress(" -> Extracted #{extracted_count} " \
        "#{Machinery::pluralize(extracted_count, "file or tree", "files and trees")}...")
```
over
```ruby
      phrase = Machinery::pluralize(extracted_count, "file or tree", "files and trees")
      Machinery::Ui.progress(" -> Extracted #{extracted_count} #{phrase}...")
```

No super strong feelings here, but I tend to disagree.